mmpose/models/losses/multi_task_loss.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+
+from mmpose.registry import MODELS
+
+
+@MODELS.register_module()
+class MultiTaskLoss(nn.Module):
```

Comment:
I suggest we name this class 'CombinedLossWrapper' or 'MultipleLossWrapper' to show that it's actually a wrapper instead of a real loss function. And this file can be renamed 'loss_wrappers.py`.